core/drivers/imx_snvs.c

```diff
@@ -61,6 +155,12 @@ static enum snvs_security_cfg snvs_get_security_cfg(void)
 	return SNVS_SECURITY_CFG_FAB;
 }
 
+static bool snvs_is_device_closed(void)
```

Comment:
can we use this function on plat_rpmb_key_is_ready replacing the "security == SNVS_SECURITY_CFG_CLOSED"at the end of the function?

Reply:
Yes, we could. I will fix that